shell/platform/fuchsia/flutter/tests/fakes/scenic/fake_flatland.h

```diff
@@ -187,6 +193,10 @@ class FakeFlatland
     std::unordered_map<zx_koid_t, BufferCollectionBinding> buffer_collections;
   };
 
+  std::unordered_map</*transform_id=*/uint64_t,
```

Comment:
Instead of keeping a map, can you instead add this to the "FakeTransform" struct in fake_flatland_types?

This allows the usage of gmock matchers when detecting hit regions

Reply:
Done